src/App.js

```diff
@@ -1,26 +1,80 @@
 import React from 'react';
-import logo from './logo.svg';
 import './App.css';
+import 'bulma/css/bulma.css';
+import FoodBox from './components/FoodBox'
+import foods from './foods.json';
+import AddFood from './components/AddFood'
+import TodaysFoods from './components/TodaysFoods'
+import FoodForm from './components/FoodForm'
+import searchBar from './components/searchBar'
```

Comment:
components needs to be written in PascalCasing. SearchBar. This is very important, otherwise the JSX won't recognise them.